modules/appeals_api/config/routes.rb

```diff
@@ -37,9 +37,7 @@
 
   namespace :v2, defaults: { format: 'json' } do
     namespace :decision_reviews do
-      # namespace :higher_level_reviews do
-      #   get 'contestable_issues(/:benefit_type)', to: 'contestable_issues#index'
-      # end
+      get 'contestable_issues/:decision_review_type', to: 'contestable_issues#index'
```

Comment:
Am I correct that for HLR contestable issues the `benefit type` will no longer be a part of the URL, but will be a param instead?

Reply:
correct, because the endpoint will be for all of the appeal types.

Reply:
Sorry for not noticing this earlier, but is there a need to separate the decision review type? Couldn't we just ask for the benefit type and return that - or are there actual differences between what HLR vs NOD contestable issues return?

Reply:
@cilestin we filter NOD where we don't filter HLR, and there's a definite need to have separation for future change. Not to mention that the Caseflow URL requires it.